src/app/components/PageLayout/Footer.tsx

```diff
@@ -129,6 +130,7 @@ export const Footer: FC<FooterProps> = ({ scope, mobileSearchAction }) => {
                 </Typography>
               </StyledLinksGroup>
             </StyledBox>
+            <CustomFooter />
```

Comment:
Doesn't appear on mobile

Reply:
Also added to mobile.
(And added commented-out example code for proper formatting, which is a bit tricky to get right)